manager.spec.yaml

```diff
@@ -1210,10 +1229,16 @@ components:
               description: Filter based on presence of security rule
               example: true,false
               nullable: true
+            patch_access:
```

Comment:
does this metadata show if the user has access to patch OR shows just meta for the  API?  for example `patch_access:true` indicates that the api has `show_advisories=true` param

Reply:
it shows if the user has access to patch, and whether patch response was successful, if patch returns an error so there will be `false`, this indicator will work only if `show_advisories=true` param was sent, in other cases, it will be null

Reply:
@katerinapat hopefully it will make easier for UI to decide whether to include the `Advisory` column in the table or not.

Reply:
@tkasparek yeap, that's why I asked, thanks